src/bio/bio_monitor.c

```diff
@@ -59,16 +60,42 @@ collect_bs_usage(struct spdk_blob_store *bs, struct nvme_stats *stats)
 	stats->avail_bytes = spdk_bs_free_cluster_count(bs) * stats->cluster_size;
 }
 
+static enum smd_dev_type
+bio_get_xs_blobstore_type(uuid_t uuid, struct bio_xs_context *xs_ctxt)
+{
+	struct bio_xs_blobstore	 *bxb;
+	enum smd_dev_type		  st;
+
+	for (st = 0; st < SMD_DEV_TYPE_MAX; st++) {
+		bxb = bio_xs_context2xs_blobstore(xs_ctxt, st);
+		if (bxb != NULL && bxb->bxb_blobstore) {
+			if (uuid_compare(uuid,
+					 bxb->bxb_blobstore->bb_dev->bb_uuid) == 0)
+				return st;
+		}
+	}
+
```

Comment:
Print an error message when it's not found? Or you print error message on caller.

Reply:
Caller check if SMD_DEV_TYPE_MAX is returned, it will return NOEXIST error